Dynatrace/dynatrace-operator
# Fixes oneAgent pod restarts after operator version change

# Description

Currently the Operator rolls out a new daemonset every time, the operator version is changed, as we still pass:

```
  - args:
    - --set-host-property=OperatorVersion=v0.11.2 
```
    
to the oneagent, which causes a new daemonset to be rolled out, as one parameter changed.
The PR adds an additional parameter to the deployment-metadata configmap and adds this as a ENV Var to the agent pod.

## How can this be tested?

Install Operator and roll out a dynakube with a oneAgent section, upgrade to a different version of the operator and check if the agent pods did not restart


## Checklist
